android/src/main/java/com/mapbox/reactnativemapboxgl/ReactNativeMapboxGLModule.java

```diff
@@ -355,6 +365,43 @@ static WritableMap serializeOfflineRegionStatus(OfflineRegion region, OfflineReg
         return result;
     }
 
+    /*
+     * Normalizes offline region status state for the sake of parity with iOS for React Native
+     * Essentially we force Android state to be the same as iOS state for ease of cross-platform development
+     *
+     * On iOS:
+     * 0: Unknown
+     * 1: Inactive
+     * 2: Active
+     * 3: Complete
+     * 4: Invalid (iOS ONLY)
+     *
+     * On Android:
+     * 0: Inactive (Complete is inactive, AND countOfResourcesCompleted == countOfResourcesExpected)
+     * 1: Active
+     */
+    static int normalizeOfflineRegionState(OfflineRegionStatus status) {
+        int state = (int)status.getDownloadState();
+        boolean isComplete = (boolean)status.isComplete();
+
+        switch (state) {
+            case 0:
```

Comment:
I think it would be good if we made some static constants for the state so the code can be self documenting here.